docs/adr/0014-use-mux-for-routing.md

```diff
@@ -0,0 +1,54 @@
+# Use gorilla/mux for Routing
+
+We need a way to serve routes so that we can create an application with functionality
+that is useful for the front end.
+
+## Considered Alternatives
+
+* [gorilla/mux](https://github.com/gorilla/mux)
+* [gin-gonic/gin](https://github.com/gin-gonic/gin)
+* Use the Standard Library Only
+* Write our own routing
+
+## Decision Outcome
+
+* Chosen Alternative: [gorilla/mux](https://github.com/gorilla/mux)
+* We've decided that gorilla/mux is the right solution
+as it is maintained by [Gorilla Tool Kit](https://www.gorillatoolkit.org/).
+It handles routing in a way that conforms to golang best
+practices and is familiar to Truss and the application developers on this team.
+* This also lets us follow more common API patterns in
+golang as described in
+[this blog post
+](https://medium.com/statuscode/how-i-write-go-http-services-after-seven-years-37c208122831)
+
+## Pros and Cons of the Alternatives <!-- optional -->
+
+### [gorilla/mux](https://github.com/gorilla/mux)
+
+* `+` Eng team is already familiar with it
+* `+` Allows us to follow well-known golang practices
+* `+` Been tested on other projects
+* `-` We'll have to make other application decisions
```

Comment:
Kara could you help me understand what this means? I'm not clear from the text on how this choice affects decision making elsewhere -- do we have more decisions to make? Or are they the same decisions but more complex/difficult? Something else?

Reply:
This only solves routing issues, unlike gin the framework which solves more potential decisions we'll have to make.

Reply:
Agree that was unclear and I'll edit it to be more clear.